src/rules/no-promise-all.js

```diff
@@ -0,0 +1,28 @@
+function isPromiseAll(node) {
+  return (
+    node.type === 'MemberExpression' &&
+    node.object.type === 'Identifier' &&
+    node.object.name === 'Promise' &&
+    node.property.type === 'Identifier' &&
+    node.property.name === 'all'
+  );
+}
+
+function create(context) {
+  return {
+    CallExpression: node => {
+      if (isPromiseAll(node.callee))
+        context.report({
+          node,
+          message: 'Use pSettle like logic instead of promise.all'
```

Comment:
```suggestion
          message: 'Prefer Promise.allSettled or pSettle like logic instead of Promise.all'
```

Vu que `Promise.allSettled()` est maintenant  là en node 12. https://node.green/#ES2020-features--Promise-allSettled